rts/Game/Camera/SpringController.cpp

```diff
@@ -96,7 +117,7 @@ void CSpringController::MouseWheelMove(float move, const float3& newDir)
 {
 	const bool moveFast    = camHandler->GetActiveCamera()->GetMovState()[CCamera::MOVE_STATE_FST];
 	const bool moveTilt    = camHandler->GetActiveCamera()->GetMovState()[CCamera::MOVE_STATE_TLT];
-	const float shiftSpeed = (moveFast ? 2.0f : 1.0f);
+	const float shiftSpeed = (moveFast ? moveFastMult : 1.0f);
```

Comment:
This introduces a new behavior, that might be ok or not. If it's not ok you'd want to do something like `(moveFastMult*2.0f)/3.0f` so the default value of 3 translates to the previous default value of 2.

Personally I'd like a single multiplier for all but in some cases it might make sense for it to be rescaled.

Reply:
I dislike magic numbers, but you are right about this changing the behaviour, I'll rescale.

Reply:
The problem here is also that configuring the modifier to 1 would imply in a slower movefast than without movefast. It's an annoying problem

Reply:
Alternatively you can pick the lowest modifier of all (this seems to be the one) and use it as the reference for the others.

Other 'solutions' are:

 - Use a single multiplier for all
 - Configure multipliers for each circumstance (does not scale well)

Reply:
There are currently multiple different modifiers at play (see your own comment about https://github.com/badosu/spring/blob/fdb444430a16d3021206a95842f9c97837704d41/rts/Game/Camera.cpp#L658-L659, which has 0.1 and 10.0). I think the best way would be to use multiple scaling parameters (moveFastMult*springFastScale here) this way there is a single moveFastMult, but if a different case needs rescaling, it can be done by config.